cime/scripts/Tools/Makefile

```diff
@@ -292,6 +292,8 @@ ifeq ($(strip $(USE_KOKKOS)), TRUE)
   CPPDEFS += -DUSE_KOKKOS_KERNELS
   include $(INSTALL_SHAREDPATH)/Makefile.kokkos
   SLIBS += $(KOKKOS_LIBS)
+  CFLAGS += $(KOKKOS_CXXFLAGS)
```

Comment:
I forgot: does CIME then use CFLAGS to populate CPPFLAGS? Is that why you stuffed KOKKOS_CXXFLAGS in CFLAGS?

Reply:
Yes:

```
CFLAGS+=$(CPPDEFS)
CXXFLAGS := $(CFLAGS)
```